docs/README.md

```diff
@@ -34,10 +34,16 @@ An asynchronous Python API wrapper meant to replace discord.py
 
 Use the following command to install Pincer into your Python environment:
 
-```bash
+```sh
 pip install pincer
 ```
 
+To install our version with Aiohttp Speedup do:
```

Comment:
It says "Aiohttp Speedup" yet `aiodns` is the only thing that speeds up `async` related things. 
`Brotli` is for data compression, `cchardet` detects the character encoding, and `orjson` works with JSON.
Apologies if I am naïve, but how do these packages speed it up?

Reply:
cchardet is a faster alternative to charset-normalizer Brotin as you said is data ``compression`` orjson is the fastest json library for python, aiohttp in aiohttp[speedups] ( https://docs.aiohttp.org/en/stable/#library-installation )